BDArmory/Control/MissileFire.cs

```diff
@@ -2995,7 +2995,7 @@ IEnumerator AllCMRoutine(int count)
             for (int i = 0; i < count; ++i)
             {
                 DropCMs((int)(CMDropper.CountermeasureTypes.Flare | CMDropper.CountermeasureTypes.Chaff | CMDropper.CountermeasureTypes.Smoke));
-                if (i < count - 1) // Don't wait on the last one.
+                if (i <= count) // Don't wait on the last one.
```

Comment:
This one shouldn't be changed. Its purpose was to not wait after dropping the last CM.

Reply:
fixed